development/index.md

```diff
@@ -38,40 +52,126 @@ There are three types of issue used for planning work:
+by the CTO. This meeting includes 2 parts:
+
+ 1. A retrospective on the last release. This should capture feedback on what
+    went well and what could be improved and generate action items to act on the
+    feedback.
+ 2. Kicking off the next release. The involves:
+    1. The Product Owner describing the themes, goals and priorities of the new release
+    2. Reviewing the [Development Board](https://github.com/orgs/flowforge/projects/1/views/1)
+       for the new milestone.
+       High-priority items are assigned owners so that everyone has something to
+       get started on.
+    3. Identifying the release manager - the person who will co-ordinate things
+       on release day.
+         - Once identified, they take the action to create the next release issue
+           using the [Release Template](https://github.com/flowforge/admin/issues/new?assignees=&labels=&template=release.md&title=Release%3A)
+           in the `flowforge/admin` repo.
 
 ### What to work on
 
-For each release a selection of epics is being worked on. When an epic assigned
-to a milestone there will be developments to complete it, however it might not
-be fully completed and closed in the assigned milestone.
+Work comes from two places:
 
-Epics are assigned to one or more individuals responsible for executing the work
-and keeping the epics up-to-date.
+ - Items assigned for Design/Estimation on the Product Board
+ - Items from the ToDo status on the Development Board
 
-Stories and tasks are assigned by the assignee of the associated epic, depending
-on how the work is distributed.
+It is important we make progress on both sides to ensure a steady velocity across
+milestones.
+
+Naturally there will be items that crop up unexpectedly and have to be dealt with
+pragmatically. For example, we may realise an item is needed for the current release
+that needs to be expedited through the process. We should remain flexible in how
+we work.
+
+When picking up an unassigned item assign it to yourself and move it to the
+**In Progress** state (for Development items) or the appropriate Product board state.
 
 #### Milestones are overscheduled
 
-As epics and stories are assigned to milestones, it's unlikely there's not
+As stories and tasks are assigned to milestones, it's unlikely there's not
 enough work to be done. On the flipside, this means that not all scheduled epics
 or stories will be completed.
 
-### Launch Lunch
+The Product Owner/CTO have overall responsibility to ensure any 'must-have' items
+are making progress in a release.
+
+### Defining Done
+
+An item should only be marked as 'Done' on the Development board when the following
+criteria are met:
+
+ - All related code changes have been merged
+ - Suitable unit/system level tests have been added
+ - Documentation has been updated
+ - Acceptance criteria identified in the Story have been met
+ - Product Owner accepts the story is complete
```

Comment:
With billing, there were additional requirements on enabling on staging and production. I don't think stories should be closed before they're deployed and configured. Can we add an extra line @knolleary ?

Reply:
Applying something to staging and production would be separate issues generally tracked in the Cloud Project

Reply:
Sure. That doesn't make the issues done, value is only delivered if customers can use it. As such it's only done when it's deployed and can be used.

Reply:
At the moment our environments are running a release version of FlowForge, therefore it can't be deployed until its in a release, once its in a release customers can use it on their own installs.
Deploying to Staging could and should be part of our testing process I believe Ben has a plan to have Staging track the main branch, so this will negate that additional step.
The intention is that our prod cloud instance should eventually get just ahead of our release process but we're not there yet.
Once we have that we can iterate this workflow